kiwi/boot/arch/arm/oemboot/suse-SLES13/config.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#================
+# FILE          : config.sh
+#----------------
+# PROJECT       : openSUSE KIWI Image System
+# COPYRIGHT     : (c) 2006 SUSE LINUX Products GmbH. All rights reserved
+#               :
+# AUTHOR        : Marcus Schaefer <ms@suse.de>
+#               :
+# BELONGS TO    : Operating System images
+#               :
+# DESCRIPTION   : configuration script for SUSE based
+#               : operating systems
+#               :
+#               :
+# STATUS        : BETA
+#----------------
+#======================================
+# Functions...
+#--------------------------------------
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# SuSEconfig
+#--------------------------------------
+suseConfig
+
+#==========================================
+# setup gfxboot
+#------------------------------------------
+suseGFXBoot openSUSE grub
```

Comment:
I realized that on PPC and s390 this part is `suseGFXBoo SLES grub` instead of openSUSE. I am just wondering if this is done in purpose or is a simple copy and paste mistake.